packages/kbn-router-to-openapispec/src/util.test.ts

```diff
@@ -117,10 +117,12 @@ describe('assignToPaths', () => {
     assignToPaths(paths, '/foo', {});
     assignToPaths(paths, '/bar/{id?}', {});
```

Comment:
I actually wonder if this type of path should translate into 2: `/bar` and `/bar/{id}`. But that's unrelated to this PR.

Reply:
++ it should, thanks for adding that check